src/03_modules.py

```diff
@@ -1,31 +1,35 @@
+import os
+import sys
 """
 In this exercise, you'll be playing around with the sys module,
 which allows you to access many system specific variables and
 methods, and the os module, which gives you access to lower-
 level operating system functionality.
 """
 
-import sys
 # See docs for the sys module: https://docs.python.org/3.7/library/sys.html
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
-
+print(sys.argv)
+print(sys.argv[0])
```

Comment:
How would you write this to print as many args are passed in? Hint: you are accessing the 0th element of sys.argv...what about 1st, 2nd, ...nth elements (if the exist?)